presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -8,6 +8,9 @@ Release 0.243
 .. warning::
     There is a bug that results in columns in the ``USING`` clause of a query to not be checked properly for column-level access control (:pr:`15333`).
 
+.. warning::
```

Comment:
was this bug introduced in 0.243 or was it a long standing issue that we just discovered now?

Reply:
this was introduced in 0.243 https://github.com/prestodb/presto/pull/15304